src/audio/volume/volume_ipc3.c

```diff
@@ -149,6 +149,10 @@ int volume_init(struct processing_module *mod)
 		cd->muted[i] = false;
 	}
 
+	cd->is_same_volume = true;
+	/* all target volume are same */
+	cd->ramp_channel_counter = 1;
+
```

Comment:
I wonder if false and 0 would be more safe initialize?

Reply:
0 may cause the first startup volume failure, set to 1 means for ipc3, all target volume are same, so only ramp one channel is ok.